src/k8s-engine/k8s.ts

```diff
@@ -60,6 +62,17 @@ export interface KubernetesBackend extends events.EventEmitter {
    */
   readonly desiredPort: number;
 
+  /**
+   * RD supports switching container engines to manage images on the VM: currently moby and nerdctl.
```

Comment:
```suggestion
   * The currently active container engine used to manage images.
```
The current choices are defined by `ContainerEngine`, no need to list them here (and get outdated).

Reply:
done